packages/design-tokens/src/scss/generated/_typography.scss

```diff
@@ -149,6 +149,99 @@ $body-xsmall-text-regular: (
     ),
 ) !default;
 
+$display-large-text: (
+    mobile: (
+        font-family: "'Nunito Sans', sans-serif",
+        font-size: 2rem,
+        font-style: normal,
+        font-weight: 800,
+        line-height: 1.3,
+    ),
+    tablet: (
+        font-family: "'Nunito Sans', sans-serif",
+        font-size: 2.375rem,
+        font-style: normal,
+        font-weight: 800,
+        line-height: 1.3,
+    ),
+    desktop: (
+        font-family: "'Nunito Sans', sans-serif",
+        font-size: 2.625rem,
+        font-style: normal,
+        font-weight: 800,
+        line-height: 1.3,
+    ),
+) !default;
+
+$display-medium-text: (
+    mobile: (
+        font-family: "'Nunito Sans', sans-serif",
+        font-size: 2rem,
+        font-style: normal,
+        font-weight: 800,
+        line-height: 1.3,
+    ),
+    tablet: (
+        font-family: "'Nunito Sans', sans-serif",
+        font-size: 2.25rem,
+        font-style: normal,
+        font-weight: 800,
+        line-height: 1.3,
+    ),
+    desktop: (
+        font-family: "'Nunito Sans', sans-serif",
+        font-size: 2.375rem,
+        font-style: normal,
+        font-weight: 800,
+        line-height: 1.3,
+    ),
+) !default;
+
+$display-small-text: (
+    mobile: (
+        font-family: "'Nunito Sans', sans-serif",
+        font-size: 1.5rem,
+        font-style: normal,
+        font-weight: 800,
+        line-height: 1.3,
+    ),
+    tablet: (
+        font-family: "'Nunito Sans', sans-serif",
+        font-size: 1.5rem,
+        font-style: normal,
+        font-weight: 800,
+        line-height: 1.3,
+    ),
+    desktop: (
+        font-family: "'Nunito Sans', sans-serif",
+        font-size: 1.5rem,
+        font-style: normal,
+        font-weight: 800,
+        line-height: 1.3,
+    ),
+) !default;
+
+$handwritten-caption: (
+    mobile: (
+        font-family: "'Caveat', sans-serif",
```

Comment:
⚠️ Caveat is hand-written font, so `monospace` as a fallback would be better.
Anyway, in Figma it's not possible to define fallback fonts, so we should find another solution...